src/providers/MeetingProvider/index.tsx

```diff
@@ -12,11 +12,17 @@ import { LocalVideoProvider } from '../LocalVideoProvider';
 import { FeaturedVideoTileProvider } from '../FeaturedVideoTileProvider';
 import { LocalAudioOutputProvider } from '../LocalAudioOutputProvider';
 import { ContentShareProvider } from '../ContentShareProvider';
+import { LogLevel } from 'amazon-chime-sdk-js';
+
+interface Props {
+  logLevel?: LogLevel;
+  children: ReactNode;
```

Comment:
`React.FC` includes children by default, so I don't think you should need to include that here

Also can you update the MeetingProvider/MeetingManager docs with this change?

Reply:
Got it, updated the docs as well. Thanks!